<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlaybackDisplay</class>
 <widget class="QFrame" name="PlaybackDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>486</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: black;
color: white;</string>
  </property>
  <widget class="QLabel" name="SpeedLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>420</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Speed: 75%&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="playButton">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>430</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: White;
background-color: black;
border: 1px solid White;</string>
   </property>
   <property name="text">
    <string>⏯</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>430</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: White;
background-color: black;
border: 1px solid White;</string>
   </property>
   <property name="text">
    <string>◀</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>430</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: White;
background-color: black;
border: 1px solid White;</string>
   </property>
   <property name="text">
    <string>▶</string>
   </property>
  </widget>
  <widget class="QSlider" name="songSlider">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>400</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SizeHorCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove::horizontal{ 
		height: 6px; 
	

	     border-radius:6px;
		border: 1px solid white;
 } 


QSlider::handle:horizontal{ 
		width:  10px; 
		height: 10px; 
		margin-top: -20px; 
		margin-left: 0px; 
		margin-bottom: -20px; 
		margin-right: 0px; 
		background: white;
		border: 1 px white;
}

QSlider::sub-page:horizontal{
	   background:rgba(80,166,234,1);
}


</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QFrame" name="Menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>35</y>
     <width>131</width>
     <height>155</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QGroupBox" name="">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="openFolderButton">
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="text">
          <string>Open</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="recentButton">
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="text">
          <string>Recent</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exitButton">
         <property name="styleSheet">
          <string notr="true">border: none;</string>
         </property>
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="nextPage">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>316</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;</string>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="SongDisplay">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>360</y>
     <width>641</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Name of the song&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="previousPage">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>316</y>
     <width>41</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;</string>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="verticalGroupBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>35</y>
     <width>441</width>
     <height>311</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;
text-align: left;</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QPushButton" name="song_0">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="song_1">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_9">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="song_2">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_7">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="song_3">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_10">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="song_4">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="line_6">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="song_5">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>1</y>
     <width>641</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; font-style:italic;&quot;&gt;Wavelow&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QSlider" name="SpeedSlider">
   <property name="geometry">
    <rect>
     <x>35</x>
     <y>447</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SizeHorCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove::horizontal{ 
		height: 6px; 
	

	     border-radius:6px;
		border: 1px solid white;
 } 


QSlider::handle:horizontal{ 
		width:  10px; 
		height: 10px; 
		margin-top: -20px; 
		margin-left: 0px; 
		margin-bottom: -20px; 
		margin-right: 0px; 
		background: white;
		border: 1 px white;
}

QSlider::sub-page:horizontal{
	   background: White;
}


</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>75</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>447</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>SizeHorCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QSlider::groove::horizontal{ 
		height: 6px; 
	

	     border-radius:6px;
		border: 1px solid white;
 } 


QSlider::handle:horizontal{ 
		width:  10px; 
		height: 10px; 
		margin-top: -20px; 
		margin-left: 0px; 
		margin-bottom: -20px; 
		margin-right: 0px; 
		background: white;
		border: 1 px white;
}

QSlider::sub-page:horizontal{
	   background: Yellow;
}


</string>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="volumeLabel">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>420</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Volume: 100%&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="shuffleButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>390</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid white;</string>
   </property>
   <property name="text">
    <string>Shuffle</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
